Sources/SwiftSyntax/generated/SyntaxBaseNodes.swift

```diff
@@ -90,49 +84,55 @@ extension DeclSyntaxProtocol {
   /// Force-upcast the current syntax node to its base node type (``DeclSyntax``).
   ///
   /// - Returns: The base node created from the current syntax node, as the node can always be upcast to its base type.
-  ///
-  /// - Note: This method overloads the general `as` method and is marked deprecated to produce a warning
-  ///         informing the user the upcast should be performed using the target base node's initializer.
   @available(*, deprecated, message: "Use `DeclSyntax.init` for upcasting")
   public func cast(_ syntaxType: DeclSyntax.Type) -> DeclSyntax {
     return DeclSyntax(self)
   }
   
 
-  /// Checks if the current syntax node can be cast to a given node type from the different base node protocol hierarchy than ``DeclSyntaxProtocol``.
+  /// Checks if the current syntax node can be cast to a given node type from a base node protocol hierarchy other
+  /// than ``DeclSyntaxProtocol``.
   ///
-  /// - Returns: `false` since the node can not be cast to the node type from different base node protocol hierarchy than ``DeclSyntaxProtocol``.
+  /// - Returns: `true` if the node can be cast, `false` otherwise.
   ///
-  /// - Note: This method overloads the general `is` method and is marked as deprecated to produce a warning,
-  ///         informing the user that the cast will always fail.
-  @available(*, deprecated, message: "This cast will always fail")
+  /// - Note: This generally does not make sense to call - if in a generic ``SyntaxProtocol`` context, use
+  /// ``DeclSyntaxProtocol`` instead. Otherwise this must be a node outside of ``DeclSyntaxProtocol``,
+  /// in which case this call will always fail.
```

Comment:
This is a little confusing to me. I’m trying to come up with a better phrasing.

Reply:
What about

```swift
  /// - Note: In most cases, this is casting a ``DeclSyntaxProtocol`` to a node that is not a ``DeclSyntaxProtocol``, which will always fail.
  ///   If the `syntaxType` argument is a generic type, constrain it to ``DeclSyntaxProtocol`` instead of ``SyntaxProtocol``.
```

Reply:
Thanks, sounds better 🙇